docs/swagger/content.yaml

```diff
@@ -250,6 +250,10 @@ definitions:
         readOnly: true
         allOf:
           - $ref: "user.yaml#/definitions/UserShort"
+      scheduled_at:
+        type: string
+        readOnly: true
```

Comment:
Is this value really `readOnly`?

Reply:
No, the line `readOnly: true` should be removed here.

Reply:
better to locate the column `scheduled_at` before `created_by`. I think it is not good to keep the column between `created_by` and `created_at`.